go.mod

```
@@ -0,0 +1,3 @@
+module github.com/enami-s/janken.app
```

すみません、気づいたらrepository名もjanken_appになってましたね。
```suggestion
module github.com/enami-s/janken_app
```

に合わせておくといいかなと思います。